postgres-appliance/scripts/upload_pg_log_to_s3.py

```diff
@@ -63,11 +64,10 @@ def upload_to_s3(local_file_path):
 
     chunk_size = 52428800  # 50 MiB
     config = TransferConfig(multipart_threshold=chunk_size, multipart_chunksize=chunk_size)
-    tags = eval(os.getenv('LOG_S3_TAGS'))
-    s3_tags_str = "&".join(f"{key}={os.getenv(value)}" for key, value in tags.items())
 
     try:
-        bucket.upload_file(local_file_path, key_name, Config=config, ExtraArgs={'Tagging': s3_tags_str})
+        bucket.upload_file(local_file_path, key_name, Config=config,
+                           ExtraArgs={'Tagging': json.loads(os.getenv('LOG_S3_TAGS'))})
```

Comment:
Is the `json.loads` really necessary here? The transformation in https://github.com/zalando/spilo/pull/1031/files#diff-e4dcf964214d58d733ae9874d4a2ab4b8e9eb99213307e1666af5dad21a326bbR791 should have converted the value to a simple "url-like"-formatted string and no further processing or conversion seems needed:

```suggestion
                           ExtraArgs={'Tagging': os.getenv('LOG_S3_TAGS', '')})
```